packages/commonwealth/server/migrations/20220926183821-add-chat-enabled.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.addColumn('Chains', 'chat_enabled', {
+      type: Sequelize.BOOLEAN,
+      allowNull: false,
+      defaultValue: true,
+    });
+  },
+
```

Comment:
can you set `axie-infinitiy` and `terra` to have chat disabled in the migration?

Reply:
🫡